src/python-frontend/type_handler.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <util/c_types.h>
+#include <nlohmann/json.hpp>
+
+class python_converter;
+
+class type_handler
+{
+public:
+  type_handler(const python_converter &converter);
```

Comment:
Can you add some documentation for these methods? You can follow the same patterns as we have on https://github.com/esbmc/esbmc/blob/master/src/goto-symex/goto_symex_state.h.